backend/src/zac/forms/api/views.py

```diff
@@ -0,0 +1,38 @@
+from drf_spectacular.utils import extend_schema
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from ..client import OpenFormsClient
+
+
+class OpenFormsFormListView(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    @extend_schema(
+        operation_id="open-forms-forms-list",
+        description="Retrieve a list of forms in open forms.",
+        responses={
+            "200": {
+                "type": "array",
+                "items": {
+                    "type": "object",
+                    "properties": {
+                        "id": {"type": "integer"},
+                        "name": {"type": "string"},
+                        "slug": {"type": "string"},
+                        "layouts": {
+                            "type": "array",
+                            "items": {
+                                "type": "string",
+                            },
+                        },
+                    },
+                },
+            }
+        },
+    )
+    def get(self, request, format=None):
+        client = OpenFormsClient()
+        definitions = client.get_forms()
+        return Response(definitions)
```

Comment:
with the suggested above changes:

```suggestion
        definitions = client.get_forms()
        serializer = self.get_serializer(instance=definitions)
        return Response(serializer.data)
```

& define

```py
def get_serializer(self, **kwargs):
    return OpenFormsProtoFormSerializer(many=True, **kwargs)
```

this ensures drf-spectacular correctly picks up the `many=True`.